app/controllers/events_controller.rb

```diff
@@ -51,7 +51,7 @@ def rsvp
     invitation = member.invitations.where(event: @event, role: 'Student').try(:first)
     invitation ||= Invitation.create(event: @event, member: member, role: 'Student')
 
-    invitation.update_attributes attending: true
+    invitation.update attending: true
```

Comment:
@notapatch can we add parenthesis to this update call so we are using it consistently across the codebase?